renovate.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["local>axonivy/renovate-config"],
+  "extends": ["local>axonivy/renovate-config:npm"],
```

Comment:
any reason why this should not relate to the npm settings?

Reply:
nop, thx